src/amo/pages/StaticPages/ReviewGuide/index.js

```diff
@@ -1,42 +1,29 @@
 /* @flow */
-import Helmet from 'react-helmet';
 import * as React from 'react';
 
-import HeadLinks from 'amo/components/HeadLinks';
-import HeadMetaTags from 'amo/components/HeadMetaTags';
-import Card from 'ui/components/Card';
+import StaticPage from 'amo/components/StaticPage';
 import translate from 'core/i18n/translate';
 import { sanitizeHTML } from 'core/utils';
 import type { I18nType } from 'core/types/i18n';
 
-import '../styles.scss';
-
 type Props = {|
   i18n: I18nType,
 |};
 
 export class ReviewGuideBase extends React.Component<Props> {
   render() {
     const { i18n } = this.props;
-
-    const title = i18n.gettext('Review Guidelines');
+    const metaDescription = i18n.gettext(`
+      Guidelines, tips, and Frequently Asked Questions to leave a review for the extensions and
```

Comment:
Why not inlining this content in the component below? (like for the `About` component)